importer.py

```diff
@@ -413,6 +413,10 @@ def _get_material_28(self, group, prim):
         if key in self.materials:
             return self.materials[key]
 
+        # Use first texture name if we have a texture and material name is default
+        if textures and self.name.startswith('default'):
+            self.name = textures[0].name
```

Comment:
I'd prefer to just keep the material name as-is, if your materials are all named "default" this is not a problem the exporter should solve IMO

Reply:
I should have clarified this more in the comment. I decided to do this because if you import an egg file which does not have materials and just textures, the importer will name the Blender material it constructs "default." [Here](https://github.com/rdb/blender-egg-importer/blob/a0fdb260e85bc9c25e2ecb860e43e85555e6bc1a/importer.py#L998)

I will still remove this though as admittedly its a rather hacky solution, and actually could inadvertently rename materials actually named default by the user. If it should be done at all it should probably be done in the EggPrimitive. But for now I'll leave it out.